### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by maoo**
- Splitted content in 2 different files
- Image cropping
- Moved recap to the top
- Minor update to README.md installation

### CODE REVIEW

File: LOCAL_RUN.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+When you deploy an application with Juju, the installation code in the charmed operator will run and set up all the resources and environmental variables needed for the application to run properly. In the case of this tutorial, we are deploying a *bundle*, which describes applications to be deployed and relationships between them.
+
+Deploy the [finos-legend-bundle](https://github.com/finos/finos-legend-bundle) using `juju deploy finos-legend-bundle`.
+
+In another terminal, you can check the deployment status and the integration code running using `watch --color juju status --color`.
+
+You'll notice that the Unit `finos-legend-gitlab-integrator-k8s/0` will get to `blocked` status; this is expected, as you'll need to [Setup and Configure GitLab](#Setup-and-Configure-GitLab).
+
+## Setup and Configure GitLab
+To run Legend, you need to either run a GitLab instance somewhere, or use GitLab.com; the type of installation really depends on user's requirements, there is a secion in `LOCAL_GITLAB.md` that talks about that (TODO).
```
Comment (balbirthomas) [inline LOCAL_RUN.md:86]: By `LOCAL_GITLAB.md` do you mean Gitlab that is not on the same host running the charmed legend application but some where else within a network reachable by the charmed applications. Note the above instructions on deploying a Gitlab are for on the same host as the charmed applications.

File: LOCAL_RUN.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+If this is your first experience with Legend, we suggest you starting with GitLab.com; you simply need to create an account and you're good to go. If, instead, you're interested to test a Legend deployment with a local GitLab, please follow instructions on `LOCAL_GITLAB.md`.
+
+To configure GitLab with Legend, you'll need a [GitLab Access Token](LOCAL_RUN.md):
+```bash
+$ juju config finos-legend-gitlab-integrator-k8s access-token="<GITLAB_ACCESS_TOKEN>"
+```
+
+`GITLAB_ACCESS_TOKEN` is what we've generated earlier; you can add `gitlab-host=<GITLAB_HOST>` to the command, if you want to point to the host/DNS/IP of another GitLab instance.
```
Comment (balbirthomas) [inline LOCAL_RUN.md:95]: I worry "another GitLab instance" may mislead here. Should we emphasise that the access token must be valid for the specific Gitlab instance pointed to by `gitlab-host=`.

**Review by balbirthomas — APPROVED** [review]
Awesome.

File: LOCAL_RUN.md (hunk 1 above)
Comment (maoo) [reply LOCAL_RUN.md:86]: that's a good point @balbirthomas 

In `LOCAL_GITLAB.md` I did a brutal copy/paste and didn't validate the content, but I agree that it should be simply renamed `GITLAB.md` for now.

File: LOCAL_RUN.md (hunk 2 above)
Comment (maoo) [reply LOCAL_RUN.md]: Text updated

**Pull request merged**